### ISSUE TITLE
Assimilation Mode

### ISSUE THREAD

**Issue by Ymabert** [Anuken/Mindustry-Suggestions#2953]
### Describe the content or mechanics you are proposing.

it would be great to review the Assimilation mode, for those who don't know what it is, it was a free-for-all mode, in a giant map and at the beginning you had a small sector of the map and a core available, with some defense, in case the other players have already started the game for a while and therefore may have troops to attack you with;  when it happened to enter a game in which there are some players who had the evolved village if they destroyed you do not get lost but simply were assimilated into his team, in practice if you know a minimum play it is the most fun mode ever, precisely because you do not it is never lost, not entirely.  I would like to know if it is possible to implement Assimilation again in multiplayer.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

if Assimilation returned the server with this mode would always be full of players and many OGs that stopped playing would certainly return to Mindustry, I remember that it was a mode that went a lot, so if you manage to implement it again, maybe even with better maps, multiplayer in my opinion would fill up more and more even with new players.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by Bottinator22** [comment]
Assimilation existing is a thing decided by specific servers, not Mindustry itself.

**Comment by RealYeet** [comment]
So like tag or capture the flag more or less

**Comment by Ymabert** [comment]
@Bottinator22 if there was a real Mindustry server, a sort of main server, it would be nice if there were all the modes, Sandbox, survival, PvP, Assimilation / FFA (these two modes alternate), competitive modes, minigames and then a lobby where you can access all these modes, only this is missing from Mindustry's multiplayer.

**Comment by OnyxFinger** [comment]
MANY low level players call it good, but high level players, especially the ones who played it's older brother FFA, don't like it as much as FFA : 
The team system is just annoying, many players uncooperative, and even if they are, they're still bad bruh how'd you fix that ?
Not only that, but ever since v6, it is extremely spawn dependant, as having a spawning location with low ressources, or players nearby will conatin you in one place for a very long time

**Comment by Ymabert** [comment]
@OnyxFinger the spawn problem will always exist, but the problem is not the resources, but who you are close to, because if you have someone close to you who has resources and you are in a corner with little chance of developing you are closed and you have to try to conquer a distant sector possibly with resources.  The problem is not knowing if the people are bad or good, as long as you have the possibility to expel them from the team and then have a core generate in a random place on the map that makes these players not assimilable / assimilable, so they cannot have help if they defeat someone and can no longer be assimilated for that game, then one the permissions to modify / destroy things would be right for him to earn them after 5 minutes of the game and with the consent of the group leader.

**Comment by Ymabert** [comment]
@itcannotbe you help to just put a random dislike to the ideas of others rather than your own opinion and argue your idea a little bit about it, because if you are against something you probably have your reasons, you probably know something about it and therefore you can also explain why you don't approve of a certain idea.

**Comment by OnyxFinger** [comment]
>  the spawn problem will always exist

in FFA ,it did not

> but the problem is not the resources, but who you are close to

both are a problem. When you spawn far enough away from someone to expand but still close, the starting cell matters a lot

**Comment by Ymabert** [comment]
@OnyxFinger apart from this positive thing is that you can delete everything and rebuild the core in another sector, obviously being a mode where it is easy to lose for these reasons or because the game has already started for 20 minutes it is right that you can assimilate / come assimilated to be able to go on more calmly.

**Comment by OnyxFinger** [comment]
> @OnyxFinger apart from this positive thing is that you can delete everything and rebuild the core in another sector, obviously being a mode where it is easy to lose for these reasons or because the game has already started for 20 minutes it is right that you can assimilate / come assimilated to be able to go on more calmly.

yes but that's why it's impossible to get stuck with a bad start, and if you have to restart everything you did in a new place because you lost, you're gonna die anyway, the enemy will have way more eco than you

**Comment by Ymabert** [comment]
@OnyxFinger in fact it is a mode that is very lucky, because many times to have energy I have to use materials and Farm, others instead I use the volcanic soil which simplifies my life a lot for example, then they are modes in which only those who know how to play wins. nabbi are always the ones who at the start use all the resources to attack you, even if you are far away and in doing so I lose both them and you, many times in fact I get bored when I find people like this, so I immediately try to fight them as soon as I see them starting to build defenses to attack my village, but it always becomes an endless war and only the one who is never part of it wins.

**Comment by OnyxFinger** [comment]
> @OnyxFinger in fact it is a mode that is very lucky, because many times to have energy I have to use materials and Farm, others instead I use the volcanic soil which simplifies my life a lot for example, then they are modes in which only those who know how to play wins. nabbi are always the ones who at the start use all the resources to attack you, even if you are far away and in doing so I lose both them and you, many times in fact I get bored when I find people like this, so I immediately try to fight them as soon as I see them starting to build defenses to attack my village, but it always becomes an endless war and only the one who is never part of it wins.

noobs wouldn't be a problem in this, if the suggestion goes through, because it's local
also, they generally don't deal a lot of damage, since they're bad at the game. As for ressources, thermals didn't actually give a big advantage back then, the only real spawn inequality was having sand or not, which I totally agree is kind of dumb
